base_search_fuzzy/models/trgm_index.py

```diff
@@ -41,10 +42,10 @@ class TrgmIndex(models.Model):
         string="Index Type",
         default="gin",
         required=True,
-        help='Cite from PostgreSQL documentation: "As a rule of thumb, a '
-        "GIN index is faster to search than a GiST index, but slower to "
-        "build or update; so GIN is better suited for static data and "
-        'GiST for often-updated data."',
+        ondelete={"gin": "set default", "gist": "set default"},
+        help="Cite from PostgreSQL documentation: GIN indexes are "
+        "the preferred text search index type."
+        "See: https://www.postgresql.org/docs/13/textsearch-indexes.html",
```

Comment:
not 13 in URL, should be current - otherwise we need to constantly update.

`https://www.postgresql.org/docs/current/textsearch-indexes.html`